css-three-column-layout/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-third">
+          <div class="circle">
+            <img src="images/merge.png" alt="merge image">
+          </div>
+        </div>
+        <div class="column-third">
+          <div class="circle">
+            <img src="images/cloud-computing.png" alt="cloud computing">
+          </div>
+        </div>
+        <div class="column-third">
+          <div class="circle">
+            <img src="images/discussion.png" alt="discussion">
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <section>
```

Comment:
`div.row` and `section` are basically doing the same thing with `display: flex` we can get rid one